### PR TITLE
Cleanup changelog

### PR DESCRIPTION
**Pull request by aG0aep6G**
Removing old entries, fixing a link.

### CODE REVIEW

File: changelog.dd [hunk 1]
```diff
@@ -6,28 +6,13 @@ $(COMMENT Pending changelog for 2.071. This will get copied to dlang.org and
 
 $(BUGSTITLE Library Changes,
 
-$(LI $(RELATIVE_LINK2 json-encode-control-characters, `std.json` encodes string control characters.))
-$(LI The package $(STDMODREF experimental_ndslice, std.experimental.ndslice) was added.
-It is also available as part of the $(LINK2 https://github.com/DlangScience/mir, Mir library).)
-$(LI Default LogLevel of FileLogger was changed to LogLevel.all.)
-$(LI Access to the internal `ptr` and `len` fields of $(XREF bitmanip, BitArray) was deprecated.)
-$(LI $(XREF experimental.allocator.gc_allocator, GCAllocator.goodAllocSize)
+$(LI $(REF GCAllocator.goodAllocSize, std,experimental,allocator,gc_allocator)
```
Comment (schveiguy) [inline changelog.dd]: You sure this is how it is supposed to be? This file doesn't get built by @CyberShadow's autotester, so it's hard for me to know whether this works.

I find it hard to believe that this needs 5 parameters.

Comment (aG0aep6G) [inline changelog.dd]: > You sure this is how it is supposed to be? This file doesn't get built by @CyberShadow's autotester, so it's hard for me to know whether this works.

I've built it with `dmd ../dlang.org/{std,html,dlang.org,macros}.ddoc -Dfchangelog.html changelog.dd`. The page doesn't look pretty, but one can see that it generates this href:

    ../phobos/std_experimental_allocator_gc_allocator.html#GCAllocator.goodAllocSize

Which looks ok. Though I've just noted that it seems to miss a dot in front of "GCAllocator". But that must be fixed in the REF macro, I think.

> I find it hard to believe that this needs 5 parameters.

Well, the module hierarchy is that deep and REF takes the segments individually, as it needs to join them differently two times: once with dots for the name, once with underscores for the URL.

Comment (schveiguy) [inline changelog.dd]: ok.

BTW, I totally understand why very few people do anything for ddoc improvements. I have no idea how to correctly use the various macros.

This LGTM.

Comment (aG0aep6G) [inline changelog.dd]: > But that must be fixed in the REF macro, I think.

https://github.com/D-Programming-Language/dlang.org/pull/1223

Comment (JakobOvrum) [inline changelog.dd]: The `REF` macro supersedes a number of less generic macros for cross-referencing. It is used correctly here.

**Comment by JakobOvrum** [comment]
LGTM, but I have no idea how the new changelog system works. Some time ago I looked around for any info on it to no avail. It'd be a nice development if anyone with a grip on how it should be used would put it in CONTRIBUTING.md or on the Wiki.

**Comment by CyberShadow** [comment]
AIUI, before a release, the release manager (Martin) essentially copies the contents of each component's changelog and pastes them in the main D changelog on the dlang.org repo.

**Comment by schveiguy** [comment]
@CyberShadow that is correct

**Comment by aG0aep6G** [comment]
Ping? Any problems with this?

**Comment by schveiguy** [comment]
I'm gonna go ahead and pull this, It's not even used until the next full release, so if there's problems they can be worked out.

Thanks.

**Comment by schveiguy** [comment]
Auto-merge toggled on

**Pull request merged**